py_bentoctl/auth_helper.py

```diff
@@ -264,6 +381,74 @@ def create_grafana_client_if_needed(token: str) -> None:
             f"    Please set BENTO_GRAFANA_CLIENT_SECRET to {client_secret} in local.env and restart Grafana",
             attrs=["bold"],
         )
+        return grafana_client_kc_id
+
+    def create_grafana_client_roles_if_needed(token: str, client_id: str) -> Optional[dict]:
+        role_representations = {}
+        for role_name in ["admin", "editor", "viewer"]:
```

Comment:
make this list a constant-named tuple since it's re-used in 2 other places

Reply:
done